src/Nordea/Components/Text.elm

```diff
@@ -211,7 +221,6 @@ variantStyle variant =
                 , fontWeight (int 500)
                 , fontSize (rem 3)
                 , lineHeight (rem 3.5)
-                , margin zero
```

Comment:
Any reason all these `margin zero` must be set for the text styles? 🤔  @fredrikloberg

Reply:
Generic components like these dont know how and where they are going to be used, so having a default margin zero makes them easier to use.`h` and `p` tags have all kinds of margins by default

Reply:
The idea is to add the margins you need when you use the component

Reply:
Aah, okay. In the cases we want to add a marging we'll have to use `|> Css.important`. Do you know if there is any way to say that margin is zero unless anything else is specified? So we won't have to use the `important` style? 😊

Reply:
I guess the `Html.span` don't need the `margin zero` property

Reply:
Ye, overriding styles is a problem with elm-css :/ Maybe we can set the zero margins with a less specific css selector

Reply:
The span elements dont need it, but we thought it was nice to just add it to every variant because the actual html tag may be changed depending on the context